wpa_supplicant/wpa_cli_cmds.c

```diff
@@ -0,0 +1,3695 @@
+/*
```

Comment:
FYI, This change to split generic commands in wpa_cli.c in to wpa_cli_cmds.c also was planned to be submitted upstream, but haven't gotten around to that.